app/src/main/java/ai/elimu/crowdsource/ui/authentication/SignInActivity.java

```diff
@@ -41,30 +48,51 @@
 /**
  * Prompts the Contributor for access to her Google account. Then stores the account details in the
  * webapp's database.
- *
+ * <p>
  * See https://developers.google.com/identity/sign-in/android/sign-in
  */
-public class SignInWithGoogleActivity extends AppCompatActivity implements View.OnClickListener {
```

Comment:
Instead of refactoring the existing `SignInWithGoogleActivity`, can we create a new Activity for the Web3 sign-in? I suggest calling it `SignInWithWeb3Activity`.

Reply:
Seems good, I'll do that!